### PR TITLE
WIP: Develop

### PR DESCRIPTION
**Pull request by bastienterrier**

### CODE REVIEW

**Comment by bastienterrier** [comment]
@EmmanuelDemey can you please review this awesome PR ? 💪  🎆

**Review by EmmanuelDemey — APPROVED** [review]

File: docs/guide/useEnvVar.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+# Environment Variables
+
+If you wanna use Environment Variables in your callback's argument, like an API key or anything else, you need to follow the next steps.
+
+## Create a `hygie-env` repo
+
+First of all, you need to create a **PRIVATE** `hygie-env` git repository at the **same level than your repo(s) using `Hygie`**.
```
Comment (EmmanuelDemey) [inline docs/guide/useEnvVar.md]: 'same lavel' ? we can have multi level in an orga ? You should say something like 'in the same organization of your repo'.

File: docs/guide/useEnvVar.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+For example, if you have the following git repositories using our solution :
+
+- /user/project1
+- /user/project2
+- /organization/project1
+- /organisation/project1
```
Comment (EmmanuelDemey) [inline docs/guide/useEnvVar.md]: two projects named project1 ?

File: docs/guide/useEnvVar.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+you need to create the two following repositories :
```
Comment (EmmanuelDemey) [inline docs/guide/useEnvVar.md]:
```suggestion
You need to create the two following repositories :
```

File: docs/guide/useEnvVar.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+::: warning
+Make sure your repo is **PRIVATE** because it will contain all the secrets you want to share with `Hygie` app.
+:::
+
+## Add a webhook
+
+Then, add a webhook with the url: `--OUR_URL--/env-var` with the `push` event.
+
+::: tip
+When you will commit your changes, we will be notice and save your data **encoded**.
```
Comment (EmmanuelDemey) [inline docs/guide/useEnvVar.md]:
```suggestion
When you will commit your changes, we will be noticed and save your data **encoded**.
```

**Pull request merged**